validators/common.ts

```diff
@@ -124,3 +125,9 @@ export const IsDecimal = (options = defaultDecimalOptions) =>
 		errorMessage: `Property must be a decimal string for ${options.locale} locale`,
 		constraints: [options],
 	});
+
+export const IsLowerCase = () =>
+	createDecorator((prop: string) => isLowerCase(prop), {
+		errorMessage: `Property must be in a string in lower cases`,
```

Comment:
I think that message should be 'Property must be a string in lower case' instead of `Property must be **in** a string in lower cases`

Reply:
@Sorikairox Yes, you're right. `Property must be a string in lower case` is grammatically right.